ipinfo/cmd_init.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/spf13/pflag"
+	"golang.org/x/term"
+)
+
+func printHelpInit() {
+	fmt.Printf(
+		`Usage: %s init [<opts>] [<token>]
+
+Options:
```

Comment:
Where are the completions for this, and the addition to the `default` command's help text?

Reply:
See @rm-Umar 's recent PR on adding the `download` subcommand to get an idea of all that's needed for adding new commands, and the expectation.